src/components/collection_voting_slab/DatasetsRequestedPanel.js

```diff
@@ -0,0 +1,145 @@
+const styles = {
+  baseStyle: {
+    fontFamily: 'Montserrat',
+    fontSize: '1.4rem',
+    fontWeight: '500',
+    color: '#333F52',
+    backgroundColor: '#F1EDE8',
+    borderRadius: '0 0 4px 4px',
+    borderBottom: '4px #646464 solid',
+    padding: '15px 25px',
+    display: 'flex',
+    flexDirection: 'column',
+    rowGap: '1.5rem'
+  },
+  heading: {
+    fontWeight: 'bold',
+    display: 'flex',
+    columnGap: '0.5rem',
+    alignItems: 'center'
+  },
+  datasetCount: {
+    color: '#747474',
+    fontSize: '1.2rem'
+  },
+  datasetList: {
+    display: 'flex',
+    flexDirection: 'column',
+    rowGap: '0.75rem'
+  },
+  link: {
+    color: '#0948B7',
+    fontWeight: '500',
+  },
+  skeletonLoader: {
+    height: '30px',
+    width: '60%'
+  }
+};
+
+export default function DatasetsRequestedPanel(props) {
+  const [filteredDatasets, setFilteredDatasets] = useState([]);
+  const [visibleDatasets, setVisibleDatasets] = useState([]);
+  const [datasetCount, setDatasetCount] = useState(0);
+  const [expanded, setExpanded] = useState(false);
+  const collapsedDatasetCapacity = 5;
+  const {bucket, dacDatasets, isLoading} = props;
+
+  const datasetsForDacInBucket = useCallback(() => {
+    const bucketElections = (!isNil(bucket) && !isNil(bucket.elections)) ? bucket.elections : [];
+    const bucketDatasetIds = flatMapDeep(bucketElections, election => {
+      return flatMapDeep(election, e =>  e.dataSetId);
+    });
+
+    return filter(dacDatasets, dacDataset => {
+      return includes(bucketDatasetIds, dacDataset.dataSetId);
+    });
+  }, [bucket, dacDatasets]);
+
+  useEffect(() => {
+    const datasets = datasetsForDacInBucket();
+    setFilteredDatasets(datasets);
+    setDatasetCount(datasets.length);
+    collapseView(datasets);
+  }, [bucket, dacDatasets, datasetsForDacInBucket]);
+
+  const collapseView = (datasets) => {
+    const datasetsHiddenWhenCollapsed = datasets.length > collapsedDatasetCapacity;
+
+    const collapsedViewDatasets = datasetsHiddenWhenCollapsed ?
+      datasets.slice(0, collapsedDatasetCapacity) :
+      datasets;
+
+    setVisibleDatasets(collapsedViewDatasets);
+  };
+
+  const SectionHeading = () => {
+    return div({style: styles.heading}, [
+      'Datasets Requested',
+      span({
+        style: styles.datasetCount,
+        isRendered: !isLoading,
+        dataCy: 'dataset-count'
+      }, [`(${datasetCount})`])
+    ]);
+  };
+
+  const DatasetList = () => {
+    const datasetRows = map(visibleDatasets, dataset => {
+      return div({style: {display: 'flex'}}, [
+        div({style: {width: '12.5%'}}, [datasetId(dataset)]),
+        div({style: {width: '75%'}}, [datasetName(dataset)])
+      ]);
+    });
+    return isLoading
+      ? div({className: 'text-placeholder', style: styles.skeletonLoader})
+      : div({style: styles.datasetList, dataCy: 'dataset-list'}, [datasetRows]);
+  };
+
+  const datasetId = (dataset) => {
+    return !isNil(dataset.alias) ? dataset.alias : '- -';
+  };
```

Comment:
The ID we show on the catalog page is labeled `dataset.datasetIdentifier` in the returned dataset we have in the DAR Collection